src/spell_checking/test/test_integration_spellchecker.py

```diff
@@ -1,14 +1,16 @@
 """Module providing functionaly needed to run integration test"""
 import unittest
 import os
+import responses
 from spell_checking.spell_checker import SpellChecker
 from text_extraction.text_extractor import TextExtractor
 
 class SpellcheckerIntegrationTests(unittest.TestCase):
     """Class containing the integrationtest between textextractor and spellchecker"""
 
+    @responses.activate

```

Comment:
Hvad gør den her linje kode?

Reply:
Brugt for at mock test der skal bruge HTTP recalls. Gør så man selv kan bestemme HTTP svaret uden at kalde selve HTTP.